src/Channels/GcmChannel.php

```diff
@@ -25,6 +25,7 @@ protected function buildData(PushMessage $message)
                 'notification' => [
                     'title' => $message->title,
                     'body' => $message->body,
+                    'icon' => $message->icon,
```

Comment:
Could you move this to a dedicated condition only for icon, like [badge has](https://github.com/Edujugon/PushNotification/blob/1bd990fb683a89519298a0326c1a47700f69bc11/src/Channels/GcmChannel.php#L34toL37) ?

Reply:
@Edujugon done bro